### PR TITLE
Fix pageUp and pageDown boot command usb key strokes

### PR DESCRIPTION
**Pull request by sylviamoss**
Fixes https://github.com/hashicorp/packer/issues/10796

### CODE REVIEW

**Comment by sylviamoss** [comment]
User confirmed the changes works https://github.com/hashicorp/packer/issues/10796#issuecomment-810194690

File: bootcommand/usb_driver.go [hunk 1]
```diff
@@ -55,8 +55,8 @@ func NewUSBDriver(send SendUsbScanCodes, interval time.Duration) *usbDriver {
 		"insert":     key.CodeInsert,
 		"home":       key.CodeHome,
 		"end":        key.CodeEnd,
-		"pageUp":     key.CodePageUp,
-		"pageDown":   key.CodePageDown,
```
Comment (azr) [inline bootcommand/usb_driver.go:59]: Does that mean that `pageUp` will stop working ?

**Review by azr — APPROVED** [review]
Had a tiny question, other than that LGTM !

File: bootcommand/usb_driver.go (hunk 1 above)
Comment (sylviamoss) [reply bootcommand/usb_driver.go:59]: They never worked. The keystrokes are lowercased before matching the map. All of the others are lowercased I don't know why I added these two camel-cased in the first place. 🤔

**Review by nywilken — APPROVED** [review]
LGTM

**Pull request merged**